sr_hardware_interface/include/sr_hardware_interface/tactile_sensors.hpp

```diff
@@ -49,13 +50,15 @@ class GenericTactileData
   GenericTactileData(bool tactile_data_valid, int sample_frequency,
                      std::string manufacturer, std::string serial_number,
                      int software_version_current, int software_version_server,
-                     bool software_version_modified, std::string pcb_version)
-          : tactile_data_valid(tactile_data_valid), sample_frequency(sample_frequency),
-            manufacturer(manufacturer), serial_number(serial_number),
-            software_version_current(software_version_current),
-            software_version_server(software_version_server),
-            software_version_modified(software_version_modified),
-            pcb_version(pcb_version)
+                     bool software_version_modified, std::string pcb_version) :
```

Comment:
in general please don't do cosmetic modifications in the same PR where you do actual changes.